src/storage/filesystem.py

```diff
@@ -165,6 +194,7 @@ def flush(self):
         self._load()
 
     def clean(self):
+        # pylint: disable=unused-variable
         minTime = int(time.time()) - (60 * 60 * 30)
         deletes = []
         for stream, streamDict in self._inventory.items():
```

Comment:
You can do a `for _, streamDict` instead of `# pylint: disable=unused-variable`.